examples/apps/cra-demo/README.md

```diff
@@ -0,0 +1,90 @@
+# Fluid Consumer Script
+README is still WIP
+
+### 1. Install CRA
+
+```bash
+npx create-react-app my-app --template typescript
+```
+
+### 2. Install Fluid and Fluid Data Objects
+
+```bash
+cd my-app-name
+npm install @fluid-experimental/fluid @fluid-experimental/data-objects
+```
+
+### 3. Import KVpair and Fluid
+
+KeyValueDataObject will provide you with a fully scaffolded DDS to store your data and subscribe to listen for changes.
+
+Fluid gives you access to methods that will bootstrap a new Fluid container, and we use the `getContainerId` helper method to simplify the creation and sharing of multi-author, Fluid sessions.
```

Comment:
```suggestion
Fluid gives you access to methods that will bootstrap a new Fluid container, and we use the `getContainerId` helper method to simplify the creation and sharing of multi-author Fluid containers.
```

I don't know if my change is accurate, but we don't really define "session" anywhere.